File: 168.ExcelSheetColumnTitle/Excel.java
class Solution {
    public String convertToTitle(int n) {
        String ans = "";
        while (n > 0) {
            n -= 1;
            ans = String.valueOf((char)(n % 26 + 'A')) + ans;
            n /= 26;
        }

        return ans;
    }
}

File: 171.ExcelSheetColumnNumber/excel.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        digit = [ord(c)-ord('A')+1 for c in s]

        ans = 0
        for n in digit:
            ans = ans * 26 + n

        return ans

if __name__ == '__main__':
    print Solution().titleToNumber('A')
